overrides/extension-override.json

```diff
@@ -46,6 +46,14 @@
         "referrer": {
             "state": "enabled"
         },
+        "clickToLoad": {
```

Comment:
old version of the extension would ignore unknown keys, right?

Reply:
Yea, I think so. I used the config editor to try using the new config that included "clickToLoad" and it seemed fine.